packages/react-aria-components/src/DropZone.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ContextValue, DOMProps, SlotProps, useContextProps, useRenderProps} from './utils';
+import {DropOptions, mergeProps, useButton, useDrop, useFocusRing, useHover} from 'react-aria';
+import React, {createContext, ForwardedRef, forwardRef} from 'react';
+
+export interface DropZoneProps extends Omit<DropOptions, 'onDropActivate' | 'onInsert' | 'onRootDrop' | 'onItemDrop' | 'onReorder'>, DOMProps, SlotProps {}
```

Comment:
I think we'll want `DropZoneRenderProps` too. Check for how Button does this.

Reply:
for now, do we just want to include isHovered, isFocused, and isFocusVisible?

Reply:
Yeah, that looks right. And we can have DropZoneProps extend `RenderProps<DropZoneRenderProps>`.

Reply:
Also, I think we can omit `'getDropOperationForPoint'` here too.